package.json

```diff
@@ -78,7 +78,7 @@
     "typescript": "^4.0.3"
   },
   "engines": {
-    "node": ">= 10.12.0"
```

Comment:
why are we removing support for node 10?

Reply:
Oh, well I thought we dropped support for v10. Some of the features like EdDSA algorithms and passphrase protected keys are only supported from v12+.

Reply:
then this field should have been updated when we did that change!

Reply:
True, looks like I forgot to do that.

Reply:
It seems v10 wasn't being tested anyways, judging by the previous matrix?